src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/TextEditingTarget.java

```diff
@@ -1558,10 +1558,8 @@ else if (!fileChange.getFile().getPath().equals(getPath()))
                return;
 
             // always check for changes if this is the active editor
-            if (commandHandlerReg_ != null)
-            {
+            if (isActiveDocument())
```

Comment:
Note that `isActiveDocument()` is a helper function that does exactly this check, so this change just ensures we reuse that utility function.